app/services/db-export/export-data-files.service.js

```diff
@@ -7,42 +7,44 @@
 
 const path = require('path')
 
-const fs = require('fs').promises
-const os = require('os')
+const fs = require('fs')
+const fsp = fs.promises
```

Comment:
Thinking about it, `fsPromises` is probably more in line with how we'd normally name this?